geomstats/stiefel.py

```diff
@@ -0,0 +1,351 @@
+"""
+Stiefel manifold St(n,p),
+a set of all orthonormal p-frames in n-dimensional space,
+where p <= n
+"""
+
+import geomstats.backend as gs
+
+from geomstats.embedded_manifold import EmbeddedManifold
+from geomstats.riemannian_metric import RiemannianMetric
+
+TOLERANCE = 1e-6
+EPSILON = 1e-6
+
+
+def matrix_f(sq_mat, f):
```

Comment:
avoid single letter for variable names please, as it makes automatic replacement harder later. thanks!